vscode-dotnet-runtime-library/src/Acquisition/InstallTrackerSingleton.ts

```diff
@@ -38,15 +47,33 @@ interface InProgressInstall
 }
 
 
-export class InstallTracker
+export class InstallTrackerSingleton
 {
+    protected static instance: InstallTrackerSingleton;
+
     protected inProgressInstalls: Set<InProgressInstall> = new Set<InProgressInstall>();
     private readonly installingVersionsKey = 'installing';
     private readonly installedVersionsKey = 'installed';
 
-    public constructor(protected readonly context : IAcquisitionWorkerContext)
+    protected constructor(protected eventStream : IEventStream, protected extensionState : IExtensionState)
+    {
+
+    }
+
+    public static getInstance(eventStream : IEventStream, extensionState : IExtensionState) : InstallTrackerSingleton
     {
+        if(!InstallTrackerSingleton.instance)
+            {
+                InstallTrackerSingleton.instance = new InstallTrackerSingleton(eventStream, extensionState);
+            }
```

Comment:
nit: Indentation.

Reply:
ack I dont know why my auto indentation tool is trying to indent like that... thanks!